tests/cases/standalone/common/flow/flow_basic.result

```diff
@@ -19,12 +19,11 @@ Affected Rows: 0
 
 -- TODO(discord9): confirm if it's necessary to flush flow here?
 -- because flush_flow result is at most 1
-admin flush_flow('test_numbers_basic');
+-- SQLNESS REPLACE ADMIN\sFLUSH_FLOW\('\w+'\)\s+\|\n\+-+\+\n\|\s+[0-9]+\s+ FLOW_FLUSHED
```

Comment:
Why do we replace the name of this column?

Reply:
> Why do we replace the name of this column?

the return of flush flow is the output row count of flow which is not determinstic so remove